monitoring/cerberus/readme.md

```diff
@@ -15,6 +15,21 @@ Use Cerberus for cluster monitoring that serves a go/no-go signal for Uptime Rob
 - turn into CCM and create template for configs
 - work on dc and configmaps for custom checks
 
+### Docker Image Build (temporary)
+```shell
+# login:
+docker login -u [bcdevops_account]
+# make sure subscription entitlement certs exist:
+etc-pki-entitlement/
+rhsm-ca/
+rhsm-conf/
+
+# build and push to specific tag
+docker build . --file Dockerfile --tag [bcdevops_account]/cerberus:[lab/prod]
```

Comment:
```suggestion
docker build . --file Dockerfile --tag [bcdevops_account]/cerberus:[lab/prod] --squash
```
Squashing all the layers at the end helps keep the entitlement key out of the final image.